app/src/main/java/com/company/strengthtracker/domain/util/DataSet.kt

```diff
@@ -11,6 +11,9 @@ class DataSet(
     var xMin = Float.MAX_VALUE
     var yMax = Float.MIN_VALUE
     var yMin = Float.MAX_VALUE
```

Comment:
Do these ever change? These could probably be made val and what you'll notice about val is that if you set it in the init() block it's ok to not set initially. This will let you never need to use max min values

So val xMin: Float

init() { xMin = 5f } works

Reply:
If you did this you would also probably just need to change the maxOf minOf thing to a reduce function since you wouldn't be able to reassign